app/src/main/java/com/github/se/polyfit/ui/flow/SettingFlow.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.se.polyfit.ui.flow
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.github.se.polyfit.model.settings.SettingOption
+import com.github.se.polyfit.ui.navigation.Navigation
+import com.github.se.polyfit.ui.navigation.Route
+import com.github.se.polyfit.ui.screen.SettingsScreen
+import com.github.se.polyfit.ui.screen.settings.AccountSettingsScreen
+
+@Composable
+fun SettingFlow(modifier: Modifier = Modifier) {
+  val navController = rememberNavController()
```

Comment:
:( 
```
  val navController = rememberNavController()
```

Reply:
This one is ok :P This flow has it's own nav controller, it needs to be defined here